### PR TITLE
set wavlm mode to `train` in both `totatonet` and `sseriouss`

### PR DESCRIPTION
**Pull request by clement-pages**
By default, wavlm (more exactly wav2vec module for `sseriouss`) is in `eval` mode in both these architectures, disabling dropout for example (#1793).

For `wav2vec` (used in `sseriouss`), wavlm was set in `eval` mode in [this](https://github.com/pytorch/audio/pull/1843) `torchaudio` PR.

### CODE REVIEW

**Comment by clement-pages** [comment]
Switching wavlm mode from `eval` to `train` gives no improvement in terms of local DER as shown on the following learning curves:

- for `totatonet` trained on AMI-SDM (blue-sky = wavlm in eval mode, dark-blue = wavlm in train mode)
image

- for  `sseriouss` trained on the compound dataset (10 diarization datasets):
image
